CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+# How to contribute?
+
+First of all, thanks for taking the time to contribute! :tada::+1:
+
+Your contributions are important to make this project and event successful. 
+You can report mistakes and errors, propose a project, offer dataset and resources or create more content. 
+
+Whatever is your background, there is a way to contribute on this [GitHub repository](https://github.com/virtual-biohackathons/covid-19-bh20), 
+on our [mailing list](https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!forum/virtual-biohackathon)
+or other [related projects listed on our wiki page](https://github.com/virtual-biohackathons/covid-19-bh20/wiki).
+
+You can raise any issue relevant to this project/event publicly in the mailing list (google group), we will address and/or assess your change proposal as promptly as we
+can to make your participation easier.
+
+We have a [Code of Conduct](https://github.com/virtual-biohackathons/covid-19-bh20/blob/master/CODE_OF_CONDUCT.md) that applies to all the activities and projects involved in this event.
+For any organisation related queries or concern, you can directly reach out to the organisers Leyla Garcia (email: ljgarciaco@gmail.com), Pjotr Prins (email: pjotr.public708@thebird.nl) 
+and  Tazro Ohta (email: t.ohta@dbcls.rois.ac.jp). 
+
+**ADD MISSING DETAILS**
+
+## How can I participate in this project/event?
+
+**ADD DETAILS**
```

Comment:
Please have a look to the [GitHub wiki](https://github.com/virtual-biohackathons/covid-19-bh20/wiki) to see the existing projects. This will help you identify possible ways to contribute to. Contact, via email or slack, the community working on that project and let them know of your interest and how you would like to participate. Conversation and collaboration will start from it.